src/mscorlib/shared/System/Char.cs

```diff
@@ -26,7 +26,7 @@ public struct Char : IComparable, IComparable<Char>, IEquatable<Char>, IConverti
         //
         // Member Variables
         //
-        private char _value;
+        private char m_value; // Do not rename (binary serialization)
```

Comment:
I do not think it is necessary to rename these to allow binary serialization.

Binary serializers have to treat primitive types like char or byte in special way. They cannot depend on the fields in them. Otherwise, they would end up with infinite recursion.